content/token-swap.md

```diff
@@ -12,6 +12,7 @@
 # TL;DR
 
 - The **token swap program** is an SPL contract deployed to Mainnet and Devnet available for use by developers and protocols.
+JC: they shouldn't use those programs and instead deploy their own -- we don't support it on any of the networks
```

Comment:
@joncinque sorry I'm not sure what "those programs" refers to here

Reply:
Sorry that wasn't clear -- I meant the token-swap program deployed to devnet and mainnet.  I mention it a bit later, but those are very old deployments that probably have loads of bugs, and aren't even managed by us.  If you wanted, we could deploy an "official" one to devnet that we keep up to date for people to test with.

Reply:
Shoot...this is making me feel like I'm missing something crucial here 😬 

So does that render everything we're doing in the lesson incorrect then?